src/Symfony/Component/Lock/Store/RetryTillSaveStore.php

```diff
@@ -76,6 +78,24 @@ public function waitAndSave(Key $key)
         throw new LockConflictedException();
     }
 
+    public function saveRead(Key $key)
+    {
+        if (!$this->decorated instanceof SharedLockStoreInterface) {
+            throw new NotSupportedException(sprintf('The store "%s" does not decorate compatible store.', get_debug_type($this)));
```

Comment:
The "%s" store must decorate a "%s" store. (ShareLockStoreInterface::class)
Same below

Reply:
thanks